### DISCUSSION TITLE
I hope to use VB.NET to develop cross-platform applications in the Uno Platform. How can I do this?

### DISCUSSION THREAD

**Discussion by rrokoko** [unoplatform/uno#18292] [Q&A]
Hello! I am a VB.NET enthusiast, and I hope to use VB.NET to develop cross-platform applications in the Uno Platform, but I can't find the relevant methods. I hope to get your help. I need fairly detailed steps. Thank you!

**Comment by jeromelaban** [comment]
Thanks for the question! Uno does not explicitly support VB.NET, and the entry point of the app needs to be in C#. That being said, you can create VB.NET libraries that reference Uno Platform, though we do not have any steps around this. 

@pkar70 may have some additional comments on this, as I recall he's been working on VB.NET with Uno Platform.

**Comment by DevTKSS** [comment]
Hey @rrokoko,
I am also developing on vb.net at work and did some research, not special to uno, but might help you either way.
The information I found was from a guy, who asked this a few years ago and was targeting a wpf if i remember this correct. The answer he got was "no, not possible" initial, but then someone came up with a maybe go-able way to use it eighter way. The key words you should search in the internet are "islands", but there was a word before that... wpf islands? Dont know exactly, sorry for that. The guy (in vb.net forum) did upload a zip file with a example, but I was not allowed to download such things, so never was able to take a look into that. Maybe you are able to do so? In my understanding this is a kind of wpf host area in a application, which is able to use the (in this case) wpf framework inside of an other framework application. e.g for a CAD Helper Programm a Winforms app is not able to show you a live 3D model along my information, but a wpf one would be, so to do that it would be host area installed in this other app. If this island thing would be able for the whole app to get a uno framework app working with a vb.net logic, thats beyond my knowledge, but I would be interested in the correct answer if you researched for that :) 
Kind regards
